packages/editor/src/components/colors/utils.js

```diff
@@ -72,3 +72,19 @@ export function getMostReadableColor( colors, colorValue ) {
 		map( colors, 'color' )
 	).toHexString();
 }
+
+/**
+ * Given an array of color objects and a color value, filters the array of colors to those that contrast.
+*
+* @param {Array}   colors           Array of color objects as set by the theme or by the editor defaults.
+* @param {?string} contrastingColor A string containing the color value to compare against for a contrast.
+*
+* @return {Array} Array of filtered colors
+*/
+export function getContrastingColors( colors, contrastingColor ) {
```

Comment:
We already have a getMostReadableColor, maybe we should call this function getReadableColors.